x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/execution/sequence/SequenceFrame.java

```diff
@@ -78,24 +63,15 @@ else if (tiebreaker != null && sequence.currentTimestamp() == timestamp) {
      * Returns the first Sequence from the group that has its timestamp
      * greater than the given argument alongside its position in the list.
      */
-    public Tuple<Sequence, Integer> after(long timestamp, Comparable<Object> tiebreaker) {
+    public Tuple<Sequence, Integer> after(Ordinal ordinal) {
```

Comment:
`before` and `after` have almost the same code. Could you reuse it in a common method?